tests/wpt/web-platform-tests/fetch/content-length/resources/content-lengths.json

```diff
@@ -105,11 +105,11 @@
   },
   {
     "input": "Content-Length: 030\r\nContent-Length: 30",
-    "output": null
+    "output": 30
```

Comment:
We will need to revert these changes and mark the tests as expected failures instead. They're tests from https://github.com/web-platform-tests/wpt and we try to keep in sync with upstream, and are designed to test behaviour across all browsers.